pkg/api/customization/pipeline/cluster_pipeline.go

```diff
@@ -203,19 +229,15 @@ func (h *ClusterPipelineHandler) authuser(apiContext *types.APIContext) error {
 		return err
 	}
 
-	if authUserInput.SourceCodeType == "github" && clusterPipeline.Spec.GithubConfig == nil {
```

Comment:
This case is removed then maybe we should return error in configClusterPipelineClientSecret?

Reply:
I'll add a check here